api/manifest.yml

```diff
@@ -13,6 +13,7 @@ defaults: &defaults
     - smtp
   env:
     RUNNER: "./vcapenvwrapper --root notify-shared --root notify-api --root notify-celery --root aws --root telstra --root twilio --root smtp"
+    NEW_RELIC_CMD: "NEW_RELIC_CONFIG_FILE=newrelic.ini newrelic-admin run-program"
```

Comment:
Do they happen to have a default for the ini file or do we have to specify everywhere `newrelic.ini`?

Reply:
There doesn't seem to be a default, without the file name it just prints it out to stdout.